Settings.h

```diff
@@ -12,20 +12,22 @@ in the source distribution for its full text.
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "HeaderLayout.h"
 #include "Process.h"
 
 
 #define DEFAULT_DELAY 15
 
 typedef struct {
-   int len;
+   uint8_t len;
    char** names;
    int* modes;
```

Comment:
Why not use `MeterModeId` from `Meter.h`?

Reply:
Cause including *Meter.h* would cause an include loop and one can not forward declare enums.